test/Validate.CFG.cpp

```diff
@@ -160,9 +160,17 @@ Block& operator>>(Block& lhs, Block& successor) {
   return lhs;
 }
 
-string header =
-    "OpCapability Shader\n"
-    "OpMemoryModel Logical GLSL450\n";
+string header(SpvCapability cap) {
+  static string *shader_header = new string(
```

Comment:
No reason to leak these.
Can just make these const char* then allow the implicit return with:
```
return (cap == SpvCapabilityShader) ? shader_header : kernel_header;
```